code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -0,0 +1,327 @@
+
+/obj/machinery/shuttle_manipulator
+	name = "shuttle manipulator"
+	desc = "I shall be telling this with a sigh\n\
+		Somewhere ages and ages hence:\n\
+		Two roads diverged in a wood, and I,\n\
+		I took the one less traveled by,\n\
+		And that has made all the difference."
+
+	icon = 'icons/obj/machines/shuttle_manipulator.dmi'
+	icon_state = "holograph_on"
+
+	var/selected_menu_key = "stat"
+	var/busy
+	// UI state variables
+	var/datum/map_template/shuttle/selected
+
+	var/obj/docking_port/mobile/existing_shuttle
+
+	var/obj/docking_port/mobile/preview_shuttle
+	var/datum/map_template/shuttle/preview_template
+	var/list/templates = list()
+	var/list/shuttle_data = list()
+
+/obj/machinery/shuttle_manipulator/New()
+	. = ..()
+	update_icon()
+
+/obj/machinery/shuttle_manipulator/update_icon()
+	overlays.Cut()
+	var/image/hologram_projection = image(icon, "hologram_on")
+	hologram_projection.pixel_y = 22
+	var/image/hologram_ship = image(icon, "hologram_whiteship")
+	hologram_ship.pixel_y = 27
+	overlays += hologram_projection
+	overlays += hologram_ship
+
+/obj/machinery/shuttle_manipulator/process()
+	return
+
+/obj/machinery/shuttle_manipulator/attack_ghost(user as mob)
+	if(!is_admin(user))
```

Comment:
This is silly, the shuttle manipulator is nanoui, it should let ghosts view it without being able to press anything, and let admins press stuff, which is the default behaviour of the default topic_interaction state.